### DISCUSSION TITLE
When using pandoc to convert *.tex to .*docx, my \chapter{} are vanishing

### DISCUSSION THREAD

**Discussion by dbourrion** [jgm/pandoc#10082] [Q&A] [answered]
I'm using pandoc to convertsome very simple *.tex files to *.docx It works very well except one thing : my \chapter{} ligne doesn't appears in the resulting .*docx

To be precise : if I compile my *.tex to .*pdf, I have a nice

> Chapitre 1
> 
> This is the first line of an incredible novel

but the export with pandoc of my *.tex give me in the resulting *.docx :

> (nothing)
> 
> This is the first line of an incredible novel

Any idea to help me ?

Best regards.

**Comment by jgm** [comment]
I tried
```
 % pandoc -o my.docx -f latex
\documentclass{book}
\begin{document}
\chapter{My chapter}
hello
\end{document}
^D
```
and I got:
<img width="137" alt="Screenshot 2024-08-15 at 11 22 27 AM" src="https://github.com/user-attachments/assets/e540653b-f8b8-4a39-a471-4a2479d16346">

**Reply by dbourrion** [reply, marked as answer]
Hi there.
In fact, I never put something in \chapter{} because i'm using in general the *.pdf export, where the "automatic" "Chapter 1" appears even with  just \chapter{} 

But that's not working "out of the box" with an docx export.

This said, somebody has give me the answer : one must use pandoc --reference-doc

You can find the answer [there](https://tex.stackexchange.com/a/724456/133589).
